LayoutTests/http/tests/gamepad/gamepad-allow-attribute.https.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      Test allow attribute with "gamepad" and getGamepads() method
+    </title>
+    <script src="../resources/js-test-pre.js"></script>
+    <script>
+      testRunner?.dumpAsText();
+      testRunner?.waitUntilDone();
+
+      function waitFor(target, eventName) {
+        return new Promise((resolve) => {
+          target.addEventListener(eventName, resolve, { once: true });
+        });
+      }
+
+      // Uses '*' (all) as the permissions policy
```

Comment:
This comment is probably for the default value, so maybe move it to the first test where default is being used.